Matveibos/hometask-2/get_middle_character.rb

```diff
@@ -0,0 +1,28 @@
+#!/usr/bin/env ruby
+
+# frozen_string_literal: true
+
+def run_cli
+  loop do
+    puts 'Enter string'
+    string = gets.chomp
+    break if string == 'exit!'
+
+    if string.empty?
+      puts 'String can not be blank!'
+    else
+      puts "Middle character: #{get_middle(string)}"
+
+    end
+  end
+end
+
+def get_middle(str)
+  if str.length.even?
+    puts str[str.length / 2 - 1] + str[str.length / 2]
```

Comment:
Check this out, you use `str.length / 2` 3 times here, I think it is better to move it to the variable, if you use this so many times
Also you can use ranges to iterate with the strings in Ruby, check how you can rewrite `get_middle` function with the ranges